agent/util-scripts/pbench-postprocess-tools

```diff
@@ -174,30 +165,32 @@ for this_tool_file in `/bin/ls $tool_group_dir`; do
 			if [ $rc -ne 0 ]; then
 				error_log "screen command failed: screen -dmS $scren_name $pbench_bin/tool-scripts/$name --$action --iteration=$iteration --group=$group --dir=$dir ${tool_opts[@]}"
 			fi
-		else
-			if [ "$action" == "kill" ]; then
-				screens_to_kill=`screen -ls | grep "$screen_name" | awk '{print $1}'`
-				pids_to_kill=`echo $screens_to_kill | awk -F. '{print $1}'`
-				if [ ! -z "$pids_to_kill" ] ;then
-					echo -n "killing the following screen sessions for $name: "
-					echo "$screens_to_kill"
-					kill $pids_to_kill
-				fi
-			else
-				$pbench_bin/tool-scripts/$name --$action --iteration=$iteration --group=$group --dir=$dir "${tool_opts[@]}" &
-				pids="$pids $!"
+		elif [ "$action" == "kill" ]; then
+			screens_to_kill=`screen -ls | grep "$screen_name" | awk '{print $1}'`
```

Comment:
screen dose not need any further rpms?! we may want to use nohup ?!

Reply:
This has been in place since day 1.  It does not cause us any dependency problems and works well for now.